Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
     json (2.3.0)
-    jwt (2.1.0)
-    jwt_signed_request (2.5.1)
-      jwt (>= 1.5.0, < 2.2.0)
+    jwt (2.2.1)
+    jwt_signed_request (2.5.4)
```

Comment:
### [jwt_signed_request](https://github.com/envato/jwt_signed_request)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.5.1 → 2.5.4

[_[change-log](https://github.com/envato/jwt_signed_request/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/jwt_signed_request)_]





<details>
<summary>Commits</summary>

A change of **20** commits. See the full changes on [the compare page](https://github.com/envato/jwt_signed_request/compare/v2.5.1...v2.5.4).

These are the first 10 commits:
- (1708163) [Fix typo](https://github.com/envato/jwt_signed_request/commit/1708163a0bbab8b25a43a6b7096b6da4b539e65c)
- (906c358) [Merge pull request #39 from envato/fix-typo](https://github.com/envato/jwt_signed_request/commit/906c3588bd1fda300047731c8d15ff4675e1ab67)
- (8d5051d) [Relax restrictions on jwt dependency](https://github.com/envato/jwt_signed_request/commit/8d5051d969cb016e98f1c55d3bf1420ee0a3c88a)
- (723344b) [Merge pull request #41 from jacobat/patch-1](https://github.com/envato/jwt_signed_request/commit/723344b4d99f6b17761932d1c2f03317701ad0db)
- (793050f) [Update gem version and changelog](https://github.com/envato/jwt_signed_request/commit/793050f20d7a41507d149d96111564908b4e462f)
- (8fcd2b0) [Merge pull request #42 from envato/sd6-new-release](https://github.com/envato/jwt_signed_request/commit/8fcd2b0c1ca9167eeffe09826af01613c02f1245)
- (43dcd43) [Ensure the request body is not emptied in Faraday v1.0.0](https://github.com/envato/jwt_signed_request/commit/43dcd43b4f0a0044b5f974d6dd909d419914af48)
- (b85e6cf) [Move assignment to a separate line](https://github.com/envato/jwt_signed_request/commit/b85e6cf3099942353cc4edf9b54df355ffb93d6f)
- (0b40aec) [Merge pull request #43 from envato/fix-incompability-with-fa…](https://github.com/envato/jwt_signed_request/commit/0b40aecc3e0ab70905c5f0a6f89e4d09ed882784)
- (2d2a178) [Bump version to 2.5.3](https://github.com/envato/jwt_signed_request/commit/2d2a178e915397f6aeb03830fce16dcfafbd4af6)


</details>